scintillometry/tests/data/NGC6440E.tim

```diff
@@ -0,0 +1,64 @@
+MODE 1

```

Comment:
I guess this one is different from the other `.tim` file, but is this really needed for `scintillometry`? If so, just a few lines should suffice.